librz/analysis/arch/arm/arm_il64.c

```diff
@@ -606,7 +606,7 @@ static RzILOpEffect *branch(cs_insn *insn) {
 	}
 	RzILOpBool *c = cond(insn->detail->arm64.cc);
 	if (c) {
-		return BRANCH(c, JMP(a), NOP);
+		return BRANCH(c, JMP(a), NOP());
```

Comment:
Could you please extract these `NOP()` changes into a separate PR?

Reply:
Done: #2705